.github/workflows/test.yml

```diff
@@ -0,0 +1,66 @@
+name: tests
+
+on: [push, pull_request]
```

Comment:
workflow_dispatch might be good to addition. It allows running tests manually by triggering it from the Actions tab.

Reply:
Done